sympy/ntheory/primetest.py

```diff
@@ -110,17 +110,22 @@ def mr(n, bases):
     return True
 
 
-def isprimeE(n,b):
-	"""isprimeE(n) - Test whether n is prime or an Euler pseudoprime to base b."""
-	if (not mr(n,b)): return False
+def isprimeE(n, b):
+    """isprimeE(n ,b) - Test whether n is prime or an Euler pseudoprime to base b."""
+	if (not mr(n, b)): return False
```

Comment:
Please make the return on its own line here and below. This makes code coverage easier to see/verify.

Reply:
Sure Sir !
Done